### PR TITLE
Add Popup

### PR DESCRIPTION
**Pull request by TiteLunda**
This PR contains:

1.Display of Work Section Using JavaScript.
2.Add of Popup Window on each cards in Works Section.
3. Add of JavaScript to implement the Popup Window.

Thanks a lot for reviewing my project. Feel free to leave me a comment

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have the javascript linters setup, kindly add the linters so that it can help you maintain industry's standard. You can find it [here](https://github.com/microverseinc/linters-config/tree/master/javascript)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -18,4 +18,233 @@ mobileLinks.forEach((link) => {
...
+  overlay.classList.add('active');
+}
+
+openMobilePopUp.forEach((button) => {
+  button.addEventListener('click', () => {
+    const modal = document.querySelector(button.dataset.modalTarget);
+    openModal(modal);
+  });
+});
+
```
Comment (aamirkhan2478) [inline script.js:238]: - [ ] Well done :clap: you did a very nice job :+1: but there are some little issues please stick to the Figma design. If you check your design popup background is not match with Figma design Kindly fix it. Check below :point_down: the comparison between your design and FIgma design :slightly_smiling_face: 

| Your Desgin   |      Figma Design      |
|---------- |:----------------:|
| Screenshot from 2023-04-19 00-58-38 |  Screenshot from 2023-04-19 00-59-08 |

File: script.js [hunk 2]
```diff
@@ -18,4 +18,233 @@ mobileLinks.forEach((link) => {
...
+                        </div>
+                    </button>
+                    </a>
+                    <a href=${cardData[0].linkToSource}>
+                    <button class="btn">
+                        <div class="popup-btn-text">${cardData[0].btn2}</div>
+                        <div class="popup-icons">
+                        <img src="./Images/Icon -GitHub.png" alt="see-live-icon">
+                        </div>
+                    </button>
```
Comment (aamirkhan2478) [inline script.js]: - [ ] Please use all button effects mentioned in the Figma design you are not using button effects on your `popup` buttons. Check below :point_down: the comparison between your design and Figma design :+1:

| Your Design   |      Figma Design      |
|---------- |:----------------:|
| Screenshot from 2023-04-19 00-53-49 |  Screenshot from 2023-04-19 00-54-16    |  

- [ ] Also, the `width` of your buttons are not matched the Figma design Kindly fix it :slightly_smiling_face:

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @TiteLunda & @uno36 , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linters passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi Team 👋, 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

[OPTIONAL] - This requirement is marked optional but **strongly** recommended, JavaScript popups are handy to display help information or to zoom in on an Image. 

- [ ] I noticed that when the popup window appears, I was able to scroll and view all other contents of the page which is not actually a good user experience for your website.  🙂

- [ ] Please kindly adjust your design so that when the popup window appears, the popup will cover the entire screen size and display over other contents. 💡

Your design
image

Figma
image



Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**